flight/targets/quanton/fw/pios_board.c

```diff
@@ -624,7 +624,7 @@ void PIOS_Board_Init(void) {
 		}
 
 		if (PIOS_I2C_CheckClear(pios_i2c_usart1_adapter_id) != 0)
-			panic(6);
```

Comment:
i still don't entirely see this one. Can you confirm that if you have a sensor plugged in but not powered it pulls the bus down? I would think an I2C peripheral would be smart enough not to do that but perhaps not.